react/week3/my-app/src/Shift.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from "react";
+import ListOfRecord from "./ListOfRecord";
+import Sub from "./Calculate";
+
+const Shift = () => {
+  const [shift, setShift] = useState({
+    eName: "",
+    sTime: "",
+    eTime: "",
+  });
+  const [record, setNewRecord] = useState([]);
+  const [search, setSearch] = useState([]);
```

Comment:
search functionality can be breakdown in other component , one component should ideally do only one functionality